include/fmt/format.h

```diff
@@ -608,7 +618,7 @@ template <typename Char>
 class null_terminating_iterator;
 
 template <typename Char>
-FMT_CONSTEXPR_DECL const Char *pointer_from(null_terminating_iterator<Char> it);
+FMT_CONSTEXPR const Char *pointer_from(null_terminating_iterator<Char> it);
```

Comment:
Why is this changed?

Reply:
That's a minor improvement. I just saw that it's a function, so `FMT_CONSTEXPR` should be used. IMO this doesn't pollute the review too much, so I allowed myself to change it.

Reply:
I think `FMT_CONSTEXPR_DECL` is more appropriate here because `FMT_CONSTEXPR` expands to `inline` and this is not a definition (and, more abstractly, this is a declaration).

Reply:
+1, will revert.